src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -109,6 +109,9 @@ function EmojiPickerMenu({forwardedRef, onEmojiSelected, activeEmoji}) {
         initialFocusedIndex: -1,
         disableCyclicTraversal: true,
         onFocusedIndexChange,
+        disableHorizontalKeys: isFocused,
+        // We need to pass isActive, we can pass true without checking visibility of the component because if the popover is not visible this picker won't be mounted
```

Comment:
```suggestion
        // We pass true without checking visibility of the component because if the popover is not visible this picker won't be mounted
```